src/rastervision/utils/files.py

```diff
@@ -56,6 +57,22 @@ def sync_dir(src_dir, dest_uri, delete=False):
     subprocess.run(command)
 
 
+def _is_raster(uri, s3_test=False):
+
+    if s3_test:
+        uri = uri.replace('s3://', '/vsis3/')
+
+    try:
+
+        rasterio.open(uri)
+
+    except:
+
+        return False
+
```

Comment:
Format nitpick: Spacing should match other code blocks.

Reply:
oops.